src/main/java/svc/data/citations/datasources/rejis/models/RejisCaseList.java

```diff
@@ -0,0 +1,25 @@
+package svc.data.citations.datasources.rejis.models;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RejisCaseList {
+
```

Comment:
REJIS may use capitalized variables but we don't, please utilize @JsonProperty to map their name to the names that make sense in our app. This could be more than just casing if you think a variable makes sense with a different name